src/components/dashboard/ActionBarChart.js

```diff
@@ -83,39 +75,57 @@ class ActionBarChart extends PureComponent {
           };
         });
 
+    return (
+      <ResponsiveContainer width="100%" height="100%">
+        <BarChart
+          width="100%"
+          height="100%"
+          data={data}
+          margin={{
+            top: 5,
+            right: 30,
+            left: 20,
+            bottom: 5,
+          }}
+        >
+          <CartesianGrid strokeDasharray="3 3" />
+          <XAxis dataKey="space" />
+          <YAxis
+            label={{
+              value: t('Action Count'),
+              angle: -90,
+              position: 'insideLeft',
+            }}
+          />
+          <Tooltip />
+          <Legend />
+          <Bar name={t('Action Count')} dataKey="count" fill={primary[type]} />
+        </BarChart>
+      </ResponsiveContainer>
+    );
+  };
+
+  render() {
+    const { spaces, t, actions } = this.props;
+
+    if (!spaces || !actions) {
```

Comment:
Can this lead to infinite loading for an empty desktop application? Or one that hasn't been used?